source/arm11/open_agb_firm.c

```diff
@@ -762,11 +770,16 @@ Result oafInitAndRun(void)
 			// Adjust the path for the save file and get save type.
 			gameCfg2SavePath(filePath, g_oafConfig.saveSlot);
 			u16 saveType;
-			if(g_oafConfig.useGbaDb || g_oafConfig.saveOverride)
+			if(g_oafConfig.saveType != (u8)-1)
```

Comment:
Same here using -1.

Reply:
Similar reasoning to above, I feel like -1 should be the value supplied instead of 0xFF. While either one would work in this instance, I feel like (u8)-1 should be used to help make it clear that this is the value we are searching for